src/slices/filters.ts

```diff
@@ -6,14 +6,35 @@ import { areFiltersEqual } from '../utils/filter-utils';
 
 const initialState: FiltersState = {};
 
+/**
+ * The payload for updating a facet option.
```

Comment:
@tmeyer2115 Do we want full internal documentation as well? I was under the impression the documentation for this item was focused on the public part of the API. Many of our internal models are well explained by their names, and I am dubious of the benefits of maintaining full internal documentation. Clean Code recommends avoiding mandated internal javadocs

Reply:
I'm in favor of some internal docs. If a method or model is adequately explained by its name, I'd say we don't need JSDocs. But, if we feel some additional clarification is needed, especially for internal classes or methods, I think we should add internal docs.

Reply:
Obviously, the ideal is what Fancher would call "self-documenting code", but that's not always possible for a method or class of suitable complexity.

Reply:
@cea2aj and @nmanu1, given the time constraints we're up against, I'm fine if we hold off on any internal docs. Getting API-Extractor set up and all the public facing things documented is more important.

Reply:
Sounds good. I've removed the redundant internal docs and will hold off on adding any more